src/api/cli/integration/integration_test.go

```diff
@@ -2184,3 +2144,119 @@ func checkWalletEntriesAndLastSeed(t *testing.T, w *wallet.Wallet) {
 	require.True(t, ok)
 	require.Equal(t, lastSeed, hex.EncodeToString(newSeed))
 }
+
+// TestLiveGUIInjectTransaction does almost the same procedure as TestCreateAndBroadcastRawTransaction.
+// The only difference is we broadcast the raw transaction throught the gui /injectTransaction api.
+func TestLiveGUIInjectTransaction(t *testing.T) {
+	if !doLive(t) {
+		return
+	}
+
+	doLiveEnvCheck(t)
+
+	if !doWallet(t) {
+		return
+	}
+
+	c := gui.NewClient(nodeAddress())
+	// prepares wallet and confirms the wallet has at least 2 coins and 8 coin hours.
+	w, totalCoins, _ := prepareAndCheckWallet(t, 2e6, 2)
```

Comment:
check for 8 coin hours

Reply:
2 coin hours are enough, burn 1 in first case, and burn the remaining 1 in second case.

Reply:
oh true. 
the comment should be updated to say 2 coin hours

Reply:
Yes, I will update the comment.